test/e2e/alert.test.js

```diff
@@ -60,10 +61,9 @@ describe('vl-alert', async () => {
         await assert.eventually.isTrue(alert.isLarge());
     });
 
-    after((done) => { 
-        if (driver) {
-            driver.quit();
-        }
-        done();
+    it('Als ik een alert met actions defineer, kan ik hiervan de tekst opvragen', async() => {
+        await assert.eventually.equal(vlAlertPage.getTekstVanButtonVanCtaAlert(), 'Fout melden');
     });
+
+    after(() => driver.quit());
```

Comment:
`driver.quit` geeft een Promise terug waarop Mocha moet wachten om zeker te zijn dat alles resolved is, dus moet zijn:
`after( async() => return driver.quit());`

Reply:
Bij een arrow function gaat hij een impliciete return doen zolang je er geen block van maakt (curly brackets).